.gitignore

```diff
@@ -3,6 +3,7 @@
 /api/resources/__pycache__/
 /api/models/__pycache__/
 
+/meltingplot-frontent/build/
```

Comment:
should this be meltingplot-frontend/build/ 
but also we should be commit the build otherwise it won't render in heroku